channels/sd.m3u

```diff
@@ -9,3 +9,9 @@ http://africatv.live.net.sa:1935/live/africatv3/playlist.m3u
 http://africatv.live.net.sa:1935/live/africatv3/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="http://mahgoubsons.com/wp/wp-content/uploads/2017/10/BNTV_logo.png" group-title="",Blue Nile
 http://livecdnh2.tvanywhere.ae/hls/blue_nile/04.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Sudan TV
+http://217.182.137.206/sudantv.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Sudania 24 TV
+http://217.182.137.206/sudania24.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Sudan Drama
+http://217.182.137.206/sudan_drama.m3u8
```

Comment:
These links don't work anymore.

Reply:
Work in Microsoft Edge & streamlink

> C:\Users>streamlink hlsvariant://http://217.182.137.206/sudan_drama.m3u8 best
[cli][info] Found matching plugin hls for URL hlsvariant://http://217.182.137.206/sudan_drama.m3u8
[cli][info] Available streams: live (worst, best)
[cli][info] Opening stream: live (hls)
[cli][info] Starting player: "C:\Program Files\VideoLAN\VLC\vlc.exe"
[cli][info] Player closed
[cli][info] Stream ended
[cli][info] Closing currently open stream...
C:\Users>

Reply:
I just double-checked the links using VLC Player and streamtest.in.
<img width="952" alt="Captura de pantalla 2020-03-01 a las 13 02 58" src="https://user-images.githubusercontent.com/7253922/75623650-253e2e00-5bbd-11ea-927c-234c700aa25a.png">
<img width="739" alt="Captura de pantalla 2020-03-01 a las 13 04 02" src="https://user-images.githubusercontent.com/7253922/75623651-27a08800-5bbd-11ea-8c0b-8510a79883b0.png">

Reply:
ok removed.